### ISSUE TITLE
add support to escape char

### ISSUE THREAD

**Issue by beenotung** [ivanceras/svgbob#37]
Sometime a char is treated as diagram char when it was intended to be word char.

Example 1:
```
co-edit
single-person
```
Is rendered as:
![1](https://user-images.githubusercontent.com/6007292/57502278-9bfff700-731d-11e9-80e4-68a35f106f98.png)
The hyphen after `o` is treated as a horizontal line.

Example 2:
```
(some remark)

some text
(some remark)
some text
```
Is rendered as:
![2](https://user-images.githubusercontent.com/6007292/57502331-d79ac100-731d-11e9-8e3c-33385dc42459.png)
The open bracket between lines is rendered as a diagram which the close bracket is still rendered as text.

**Comment by ivanceras** [comment]
Yes the `-` is chained together with `o` and interpreted as line with circle endpoint.
You can escape it by using `"` ie: `"co-edit"`

**Comment by chaseleinart** [comment]
Hey there @ivanceras,
It does seem that escaping the "o character" when chained between adjacent text is problematic. when by itself, it seems to be escaped fine, but when the _o falls in the middle of a string of characters without any spaces, it doesn't seem like it can be escaped without some odd spaces being displayed.

**Input**
         "_o"                                                                                                   
        
     foo_bar_opq_id 
                                                                                            
     fo"o_"bar"_o"pq_id                                                                                                       
      

**Output**
![image](https://user-images.githubusercontent.com/11204055/62379152-616cbb80-b514-11e9-89d7-e0e93bc2b248.png)

**Comment by virtualirfan** [comment]
@chaseleinart why not just:
```
"foo_bar_opq_id"
```
instead of
```
fo"o_"bar"_o"pq_id                                                                                                       
```

**Comment by makaleks** [comment]
> Yes the `-` is chained together with `o` and interpreted as line with circle endpoint.
> You can escape it by using `"` ie: `"co-edit"`

Thanks for this tip, it is not used widely in demo. The problem is that it does not saves alignment for inherited text. For example, when I print
```
 +----------+
 |"BIOS RAM"|
 +----------+
      /\
     '  '
    /    \
+----+  +----+
|"ПО"|  |"БД"|
+--o-+  +----+
```
it results into 
![Screenshot-2019-10-21 Screenshot](https://user-images.githubusercontent.com/6663351/67196109-d8726b80-f402-11e9-8168-c7332b6165e8.png)

If I replace quotes with spaces, all is normal. But I need these escape-quotes in other places of diagram and the missed alignment seems to be a bigger problem

Thanks

**Comment by ivanceras** [comment]
This has been fixed in svgbob 0.5.5

Try this out on the [online editor](https://ivanceras.github.io/svgbob-editor/)

````
 +----------+
 |"BIOS RAM"|
 +----------+
      /\
     /  \
    /    \
+----+  +----+
|"ПО"|  |"БД"|
+--o-+  +----+

 "foo_bar_opq_id" 
                                                                                        
"(some remark)"
some text
````
